dspace/config/submission-forms.xml

```diff
@@ -295,125 +217,94 @@
+                    <input-type>textarea</input-type>
+                    <hint>Spatial characteristics of the resource.</hint>
+                    <required>Please enter a spatial characteristics of the resource.</required>
+                    <vocabulary>srsc</vocabulary>
                 </field>
             </row>
             <row>
                 <field>
-                    <dc-schema>metashare</dc-schema>
-                    <dc-element>ResourceInfo#ResourceComponentType#ToolServiceInfo</dc-element>
-                    <dc-qualifier>languageDependent</dc-qualifier>
-                    <repeatable>false</repeatable>
-                    <label>Language Dependent</label>
-                    <type-bind>toolService</type-bind>
-                    <input-type value-pairs-name="metashare_languageDependent">list</input-type>
-                    <hint>Indicate whether the operation of the tool or service is
-                        language dependent or not
-                    </hint>
-                    <required>Please indicate whether the tool is language dependent</required>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>identifier</dc-element>
+                    <dc-qualifier>citation</dc-qualifier>
+                    <repeatable>true</repeatable>
+                    <label>Bibliographic Citation</label>
+                    <input-type>textarea</input-type>
+                    <hint>A bibliographic reference for the resource.</hint>
+                    <required>Please enter a bibliographic reference(s) for the resource.</required>
+                    <vocabulary>srsc</vocabulary>
                 </field>
             </row>
             <row>
                 <field>
-                    <dc-schema>local</dc-schema>
-                    <dc-element>bitstream</dc-element>
-                    <dc-qualifier>redirectToURL</dc-qualifier>
-                    <repeatable>false</repeatable>
-                    <label>Upload file from URI (Admin only field)</label>
-                    <input-type>onebox</input-type>
-                    <hint>
-                        Please enter the full path of the file, either from the local server or an HTTP location.
-                        The maximum upload size for a file is 4GB. To upload a file larger than this limit, provide the
-                        URL of the file. Ensure the admin is aware of the URL of the bitstream file.
-                        Once the file path or URL is entered, click the 'Save' button to begin the upload.
-                        Note: The file path must be an absolute path.
-                    </hint>
-                    <required></required>
-                    <acl>
-                        policy=deny,action=read,grantee-type=user,grantee-id=*
-                    </acl>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>accessRights</dc-element>
```

Comment:
Use dc.rights like in the ZCU
https://github.com/dataquest-dev/DSpace/blob/customer/zcu-data/dspace/config/submission-forms.xml#L1214